Gopkg.toml

```diff
@@ -29,3 +29,7 @@
 [[constraint]]
   name = "sigs.k8s.io/controller-runtime"
   version = "v0.1.4"
+
+[[override]]
+  name = "gopkg.in/fsnotify.v1"
+  source = "gopkg.in/fsnotify/fsnotify.v1"
```

Comment:
Add new line.
Also why is this part of Gopkg.toml
Is this a direct dependency that we need to pin?

Reply:
This particular package + `dep ensure` has caused many people issues. Check out [this issue comment](https://github.com/golang/dep/issues/1799#issuecomment-408051454) for details.